lib/db/mysql/patches/patch-000-001.sql

```diff
@@ -0,0 +1,9 @@
+-- Create the 'dbMetadata' table.
+-- Note: This should be the only thing in this initial patch.
+
+CREATE TABLE dbMetadata (
+  name VARCHAR(255) NOT NULL PRIMARY KEY,
+  value VARCHAR(255) NOT NULL
+) ENGINE=InnoDB;
+
+INSERT INTO dbMetadata SET name = 'schema-patch-level', value = '1';
```

Comment:
huh, the patch itself is responsible for updating the patch-level? The code looks fine, and won't block this, but I'd personally expect this to be handled by the Patcher itself...